src/CategoriesPage.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Nav from './components/Nav';
+
+function CategoriesPage() {
+  return (
+    <>
+      <Nav />
+      <div>CategoriesPage</div>
+    </>
```

Comment:
- [ ] the categories page should display only a button with text "Check status"

Reply:
done sir